cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -189,12 +294,746 @@ def _generate_url_dump_tasks_annotations(self, task_id):
+                        self._upload_file(url, binary_file, self.admin)
+                    task_ann = TaskAnnotation(task_id)
+                    task_ann.init_from_db()
+                    task_ann_data = task_ann.data
+                    self.assertEqual(len(task_ann_data["shapes"]), len(task_ann_prev_data["shapes"]))
+
+    def test_api_v1_tasks_annotations_dump_and_upload_many_jobs_with_datumaro(self):
+        test_name = self._testMethodName
+        upload_format_name = "CVAT 1.1"
+        include_images_params = (False, True)
+        dump_format_names = ("CVAT for images 1.1", "CVAT for video 1.1")
+
+        for dump_format_name, include_images in itertools.product(dump_format_names, include_images_params):
+            with self.subTest(f"{dump_format_name}_include_images_{include_images}"):
+                # create task with annotations
+                images = self._generate_task_images(13)
+                task = self._create_task(tasks["many jobs"], images)
+                self._create_annotations(task, f'{dump_format_name} many jobs', "default")
+
+                task_id = task["id"]
+                data_from_task_before_upload = self._get_data_from_task(task_id, include_images)
+
+                # dump annotations
+                url = self._generate_url_dump_tasks_annotations(task_id)
+                with TestDir() as test_dir:
+                    file_zip_name = osp.join(test_dir, f'{test_name}_{dump_format_name}.zip')
+
+                    data = {
+                        "format": dump_format_name,
+                        "action": "download",
+                    }
+                    self._download_file(url, data, self.admin, file_zip_name)
+                    self._check_downloaded_file(file_zip_name)
+
+                    # remove annotations
+                    self._remove_annotations(url, self.admin)
+
+                    # upload annotations
+                    url = self._generate_url_upload_tasks_annotations(task_id, upload_format_name)
+                    with open(file_zip_name, 'rb') as binary_file:
+                        self._upload_file(url, binary_file, self.admin)
+
+                    # equals annotations
+                    data_from_task_after_upload = self._get_data_from_task(task_id, include_images)
+                    compare_datasets(self, data_from_task_before_upload, data_from_task_after_upload)
+
+    def test_api_v1_tasks_annotations_dump_and_upload_with_datumaro(self):
+        test_name = self._testMethodName
+        # get formats
+        dump_formats = dm.views.get_export_formats()
+        include_images_params = (False, True)
+        for dump_format, include_images in itertools.product(dump_formats, include_images_params):
+            if dump_format.ENABLED:
+                dump_format_name = dump_format.DISPLAY_NAME
+                with self.subTest():
+                    if dump_format_name in [
+                        "MOT 1.1", # issue #2925
```

Comment:
@MashaSS , I see that some of these problems were fixed but tests are disabled. Could you please re-check?

Reply:
https://github.com/openvinotoolkit/cvat/issues/2840

Reply:
`MOT 1.1` and `MOTS PNG 1.0` cannot be checked by datumaro. I removed old comments about this issue.